Courses/Views/Screens/HomeView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  HomeView.swift
+//  Courses
+//
+//  Created by Dmytro Churakov on 07.10.2022.
+//
+
+import SwiftUI
+
+struct HomeView: View {
+    @State var hasScrolled = false
+    
+    var body: some View {
+        ZStack {
+            Color("Background").ignoresSafeArea()
+            
+            ScrollView {
+                scrollDetection
+                featured
+                Color.clear.frame(height: 1000)
```

Comment:
@DCHRKV why here is used 1000?

Reply:
@Den-Ree , typo, corrected by `100 `

Reply:
@DCHRKV it's in general not clear why you set clear color with frame. And why you can set 1000, and then 100.

Reply:
@Den-Ree , this is my mistake due to inattention. The line was written for the test and I forgot to delete it and also duplicated it already in the right place (safeAreaInset). This line is meaningless and will be removed.